ansible/istio/tasks/delete_old.yml

```diff
@@ -0,0 +1,36 @@
+- name: Delete Addons
+  command: "{{ oc_path }} delete -f {{ istio_dir }}/install/kubernetes/addons/{{ item }}.yaml --ignore-not-found=true -n {{ istio.namespace }}"
+  ignore_errors: true
+  with_items: "{{ istio_all_addons }}"
+
+- name: Delete Addon ServiceAccounts
+  command: "{{ oc_path }} delete sa istio-{{ item }}-service-account --ignore-not-found=true -n {{ istio.namespace }}"
+  ignore_errors: true
+  with_items: "{{ istio_all_addons }}"
+
+
+- name: Delete contect from Istio Kubernetes file
+  command: "{{ oc_path }} delete -f {{ istio_dir }}/install/kubernetes/istio.yaml --ignore-not-found=true -n {{ istio.namespace }}"
```

Comment:
A problem that we will have with this command is that openshift will try to delete the namespace of istio-system (idem also for istio-auth)

Reply:
And just try to opportunistically remove things. What do you think?

Reply:
Can we transform the file to remove the lines about namespace resource ?

Reply:
Yes, that's of course doable :)